configure.ac

```diff
@@ -1075,6 +1081,7 @@ AX_BOOST_SYSTEM
 AX_BOOST_FILESYSTEM
 AX_BOOST_THREAD
 AX_BOOST_CHRONO
+AX_BOOST_PROCESS
```

Comment:
This seems like it will pull in Boost.Process (and our - in this case dead - wrapper code using it) even with `--disable-external-signer`.

Maybe not a problem since we plan to use it for other things too?